test/cri-containerd/runpodsandbox_test.go

```diff
@@ -171,230 +126,133 @@ func Test_RunPodSandbox_VirtualMemory_WCOW_Hypervisor(t *testing.T) {
+	request := getRunPodSandboxRequest(t, wcowHypervisorRuntimeHandler)
+	request.Config.Annotations = map[string]string{
+		"io.microsoft.virtualmachine.computetopology.memory.allowovercommit": "false",
 	}
-	runPodSandboxTest(t, request)
+	runPodSandboxTest(t, &request)
 }
 
 func Test_RunPodSandbox_FullyPhysicallyBacked_WCOW_Hypervisor(t *testing.T) {
 	requireFeatures(t, featureWCOWHypervisor)
 
 	pullRequiredImages(t, []string{imageWindowsNanoserver})
 
-	request := &runtime.RunPodSandboxRequest{
-		Config: &runtime.PodSandboxConfig{
-			Metadata: &runtime.PodSandboxMetadata{
-				Name:      t.Name(),
-				Namespace: testNamespace,
-			},
-			Annotations: map[string]string{
-				"io.microsoft.virtualmachine.fullyphysicallybacked": "true",
-			},
-		},
-		RuntimeHandler: wcowHypervisorRuntimeHandler,
+	request := getRunPodSandboxRequest(t, wcowHypervisorRuntimeHandler)
+	request.Config.Annotations = map[string]string{
+		"io.microsoft.virtualmachine.fullyphysicallybacked": "true",
 	}
-	runPodSandboxTest(t, request)
+	runPodSandboxTest(t, &request)
 }
 
 func Test_RunPodSandbox_PhysicalMemory_LCOW(t *testing.T) {
 	requireFeatures(t, featureLCOW)
 
 	pullRequiredLcowImages(t, []string{imageLcowK8sPause})
 
-	request := &runtime.RunPodSandboxRequest{
-		Config: &runtime.PodSandboxConfig{
-			Metadata: &runtime.PodSandboxMetadata{
-				Name:      t.Name(),
-				Uid:       "0",
-				Namespace: testNamespace,
-			},
-			Annotations: map[string]string{
-				"io.microsoft.virtualmachine.computetopology.memory.allowovercommit": "false",
-			},
-		},
-		RuntimeHandler: lcowRuntimeHandler,
+	request := getRunPodSandboxRequest(t, lcowRuntimeHandler)
+	request.Config.Annotations = map[string]string{
+		"io.microsoft.virtualmachine.computetopology.memory.allowovercommit": "false",
 	}
-	runPodSandboxTest(t, request)
+	runPodSandboxTest(t, &request)
 }
 
 func Test_RunPodSandbox_FullyPhysicallyBacked_LCOW(t *testing.T) {
 	requireFeatures(t, featureLCOW)
 
 	pullRequiredLcowImages(t, []string{imageLcowK8sPause})
 
-	request := &runtime.RunPodSandboxRequest{
-		Config: &runtime.PodSandboxConfig{
-			Metadata: &runtime.PodSandboxMetadata{
-				Name:      t.Name(),
-				Namespace: testNamespace,
-			},
-			Annotations: map[string]string{
-				"io.microsoft.virtualmachine.fullyphysicallybacked": "true",
-			},
-		},
-		RuntimeHandler: lcowRuntimeHandler,
+	request := getRunPodSandboxRequest(t, lcowRuntimeHandler)
+	request.Config.Annotations = map[string]string{
+		"io.microsoft.virtualmachine.fullyphysicallybacked": "true",
 	}
-	runPodSandboxTest(t, request)
+	runPodSandboxTest(t, &request)
 }
 
 func Test_RunPodSandbox_MemorySize_WCOW_Process(t *testing.T) {
 	requireFeatures(t, featureWCOWProcess)
 
 	pullRequiredImages(t, []string{imageWindowsNanoserver})
 
-	request := &runtime.RunPodSandboxRequest{
-		Config: &runtime.PodSandboxConfig{
-			Metadata: &runtime.PodSandboxMetadata{
-				Name:      t.Name(),
-				Uid:       "0",
```

Comment:
No idea what this is used for 🤣 but we aren't setting it anymore for a couple tests.

Reply:
from what I saw, it's used to construct sandbox name. so having it set to the same value doesn't accomplish much... we could randomize it I guess.